goodtablesio/handlers.py

```diff
@@ -0,0 +1,19 @@
+# API hadler stubs (instead of request it gets payload/task_id and returns)
+import dataset
+from . import tasks
+from . import services
+
+
+# Module API
+
+def post_task(payload):
```

Comment:
thanks, renamed - but anyway I suppose you will design API by yourself so used here names just a "random" choices to be synced with API work